src/server/pfs/server/driver.go

```diff
@@ -643,11 +628,15 @@ func (d *driver) findCommits(ctx context.Context, request *pfs.FindCommitsReques
 				}
 			}
 			commitsSearched++
-			if (foundCommits == request.Limit && request.Limit != 0) || inspectCommitResp.ParentCommit == nil {
+			if (foundCommits == request.Limit && request.Limit != 0) || inspectCommitResp.Parent == nil {
 				searchDone = true
 				return nil
 			}
-			commit = inspectCommitResp.ParentCommit
+			if inspectCommitResp.Parent != nil {
+				commit = inspectCommitResp.Parent.Pb()
+				return nil
+			}
+			commit = nil
```

Comment:
[nit] is this line necessary? I comment this out and TestFindCommits still passed. Let me know if there is any concern.